pages/_app.tsx

```diff
@@ -0,0 +1,24 @@
+import { Provider } from 'react-redux';
+import { store } from '../src/redux/store';
+import '../src/index.css';
+import { createWrapper } from 'next-redux-wrapper';
+import React, { ReactNode } from 'react';
+import { AppProps } from 'next/app';
+
+interface MyAppProps extends AppProps {
+  Component: React.ComponentType<AppProps['pageProps']>;
+}
+
+export function MyApp({ Component, pageProps }: MyAppProps): ReactNode {
+  return (
+    <Provider store={store}>
+      <Component {...pageProps} />
+    </Provider>
+  );
+}
+
+const makeStore = () => store;
+
+export const wrapper = createWrapper(makeStore);
+
+export default wrapper.withRedux(MyApp);
```

Comment:
There is an error 'You are using legacy implementation. Please update your code: use createWrapper() and wrapper.useWrappedStore().' when run dev